architecture-model-generator/modules/architecture-model-generator-core/src/main/java/io/ballerina/architecturemodelgenerator/core/generators/entrypoint/nodevisitors/FunctionEntryPointVisitor.java

```diff
@@ -118,9 +126,18 @@ public void visit(FunctionDefinitionNode functionDefinitionNode) {
                 diagnostics.add(diagnostic);
             }
 
-            functionEntryPoint = new FunctionEntryPoint(funcParamList, returnTypes,
+            String functionId = annotation != null ? annotation.getId() : functionDefinitionNode.functionName().text();
```

Comment:
Since `functionDefinitionNode.functionName().text()` is always `main` can we replace this with `MAIN` constant